ReClass.NET/Core/CoreFunctionsManager.cs

```diff
@@ -12,9 +12,10 @@ namespace ReClassNET.Core
 {
 	public class CoreFunctionsManager : IDisposable
 	{
-		private readonly Dictionary<string, ICoreProcessFunctions> functionsRegistry = new Dictionary<string, ICoreProcessFunctions>();
+		public readonly Dictionary<string, ICoreProcessFunctions> functionsRegistry = new Dictionary<string, ICoreProcessFunctions>();
```

Comment:
This private member should not be public. You can't change this atm because there is no other way, so this is just a note for me.